Lib/fontTools/pens/hashPointPen.py

```diff
@@ -33,8 +35,14 @@ class HashPointPen(AbstractPointPen):
     >    pass
     """
 
-    def __init__(self, glyphWidth=0, glyphSet=None):
+    def __init__(
+        self,
+        glyphWidth=0,
+        glyphSet=None,
+        roundFunction=partial(floatToFixedToFloat, precisionBits=14),
```

Comment:
TBH I don't think the floatToFixedToFloat behavior should be the default. Since this is specific to your use case, I think you have to enable it explicitly.

Reply:
I thought that it was useful for all cases, but of course you are right.